udata_hydra_csvapi/utils.py

```diff
@@ -0,0 +1,37 @@
+def build_sql_query_string(request_arg: list, page_size: int, offset: int = 0) -> str:
+    sql_query = []
+    for arg in request_arg:
+        argument, value = arg.split("=")
+        if "__" in argument:
+            column, comparator = argument.split("__")
+            normalized_comparator = comparator.lower()
+            if normalized_comparator == "sort":
+                if value == "asc":
+                    sql_query.append(f"order={column}.asc")
+                elif value == "desc":
+                    sql_query.append(f"order={column}.desc")
+            elif normalized_comparator == "exact":
+                sql_query.append(f"{column}=eq.{value}")
+            elif normalized_comparator == "contains":
+                sql_query.append(f"{column}=like.*{value}*")
```

Comment:
I think it should probably be case insensitive? Seems like it's the case with the actual csvapi
```suggestion
                sql_query.append(f"{column}=ilike.*{value}*")
```